packages/lib/src/project/AppProject.js

```diff
@@ -60,31 +61,29 @@ export default class AppProject extends Project {
   }
 
   // TODO: Testme
-  async changeProxyAdmin(proxyAddress, newAdmin) {
-    return this.app.changeProxyAdmin(proxyAddress, newAdmin)
-  }
-
-  // TODO: Testme
-  async createContract(contractClass, { packageName, contractName, initMethod, initArgs }) {
+  async createContract(contractClass, { packageName, contractName, initMethod, initArgs } = {}) {
     if (!contractName) contractName = contractClass.contractName
     if (!packageName) packageName = this.name
     return this.app.createContract(contractClass, packageName, contractName, initMethod, initArgs)
   }
 
-  // TODO: Testme
-  async createProxy(contractClass, { packageName, contractName, initMethod, initArgs }) {
+  async createProxy(contractClass, { packageName, contractName, initMethod, initArgs } = {}) {
     if (!contractName) contractName = contractClass.contractName
     if (!packageName) packageName = this.name
+    if (!_.isEmpty(initArgs) && !initMethod) initMethod = 'initialize'
```

Comment:
I think this check was already performed at an upper level, isn't it? If so, do you think it still make sense to have it duplicated?

Reply:
It may be done at an upper level on the CLI, but since we may want to start promoting usage of the `lib` directly, we need to have it here as well.

Reply:
Actually I scanned through the NetworkAppController, scripts/create, and commands/create, and didn't found it. It could be that I already removed it when making this change.